<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1401</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>501</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 1px solid gray; 
     border-radius: 4px; 
 } 

QGroupBox::title { 
    background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
     padding:2 13px;
 } 
</string>
    </property>
    <property name="title">
     <string>LED Control</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>191</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="0">
       <widget class="QRadioButton" name="ledVolumeRB">
        <property name="text">
         <string>Volume</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="ledWaitingRB">
        <property name="text">
         <string>Waiting</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="ledAllOneColorRB">
        <property name="text">
         <string>All One Color</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="ledDisplayDataRB">
        <property name="text">
         <string>Display Data</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="ledListeningRB">
        <property name="text">
         <string>Listening</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="ledAllOffRB">
        <property name="text">
         <string>All Off</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="ledSpeakingRB">
        <property name="text">
         <string>Speaking</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QRadioButton" name="ledAudioVoiceLocatedRB">
        <property name="text">
         <string>Auto Voice Locate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QSlider" name="ledVolumeSlider">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
     <property name="value">
      <number>8</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="ledAllColorButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="infoMessageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>871</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="sourceTabWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>641</width>
      <height>181</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="inputTab">
     <attribute name="title">
      <string>Input</string>
     </attribute>
     <widget class="QComboBox" name="inputDeviceCB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>600</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="outputTab">
     <attribute name="title">
      <string>Output</string>
     </attribute>
     <widget class="QComboBox" name="outputDeviceCB">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>600</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="recordButton">
       <property name="text">
        <string>Record</string>
       </property>
       <property name="icon">
        <iconset resource="src/micarray.qrc">
         <normaloff>:/images/record.png</normaloff>:/images/record.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Waveform" name="waveformWidget" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>250</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="LevelMeter" name="levelMeterWidget" native="true">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>310</y>
      <width>41</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="Spectrograph" name="spectrographWidget" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="vadIndicator">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>32</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="src/micarray.qrc">:/images/32px-Button_Icon_Green.svg.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1401</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuReSpeaker_Jetson_Example">
    <property name="title">
     <string>ReSpeaker Jetson Example</string>
    </property>
   </widget>
   <addaction name="menuReSpeaker_Jetson_Example"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Waveform</class>
   <extends>QWidget</extends>
   <header location="global">waveform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LevelMeter</class>
   <extends>QWidget</extends>
   <header location="global">levelmeter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Spectrograph</class>
   <extends>QWidget</extends>
   <header location="global">spectrograph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="src/micarray.qrc"/>
 </resources>
 <connections/>
</ui>
